tvtk/pyface/ui/wx/scene.py

```diff
@@ -571,6 +571,10 @@ def OnButtonUp(self, event):
     def _closed_fired(self):
         super(Scene, self)._closed_fired()
         self.picker = None
+        # Remove OnPaint handler for PaintEvent, otherwise
+        # OnPaint tries to reset the size of the nonexisting
+        # renderwindow
+        wx.EVT_PAINT(self._vtk_control, None)
```

Comment:
`_create_control` is called here https://github.com/enthought/mayavi/blob/fix-savefig-size-black-image/tvtk/pyface/tvtk_scene.py#L205  which assigns `_vtk_control`.  Unless `self._vtk_control = None` is done somewhere outside of this function (which I have not checked), we can ensure `_vtk_control` is not None?